sniffer.py

```diff
@@ -150,7 +156,23 @@ def main():
                 pkt = result.value[2:2+length]
                 if options.ti_crc:
                     pkt = ti_crc(pkt)
-                pkt = pcap.encode_frame(pkt)
+
+                # metadata format:
+                # 0. RSSI(int8)
+                # 1. Noise Floor(int8)
+                # 2. Flags(uint16)
+                # 3. PHY-specific data struct contains:
```

Comment:
Ok, I'm fine to add the timestamp to standard metadata field.

@darconeous, May I know your opinion?